### ISSUE TITLE
Suggestion: Multiple label colors 

### ISSUE THREAD

**Issue by especiallyblue** [nmalkin/plot-likert#40]
Would it be possible to implement an option that allows for using different label colors depending on the color of the bar? For instance, if I have a dark-gray bar, I want the label to be white. Likewise, if I have a light-gray bar, I want the label to be black. If I'm not mistaken the code in its current state only allows for using a single color for all labels. 

Otherwise, thank you for this great visualization tool!

**Comment by nmalkin** [comment]
Thanks for the suggestion! It looked to be a pretty straightforward improvement, so I implemented it in 9cd4c3cd26f2e0c5f3af6ae9687e09d7f76135b5

You can now use a list of colors as a value for `bar_labels_color`, instead of a single value:
```python
plot_likert.plot_likert(data, plot_likert.scales.agree, plot_percentage=True,
                        colors=plot_likert.colors.default_with_darker_neutral,
                        bar_labels=True, bar_labels_color=["snow", "white", "red", "green", "blue"]);

```

You can test it out with:  
`pip install --pre plot-likert`

Let me know how it works!

**Comment by especiallyblue** [comment]
Works beautifully, thank you!

**Comment by nmalkin** [comment]
This change is now available in the mainline version ([v0.5.0](https://github.com/nmalkin/plot-likert/releases/tag/v0.5.0)).
